packages/core/src/modules/routing/services/MessagePickupService.ts

```diff
@@ -16,14 +17,6 @@ export class MessagePickupService {
     this.messageRepository = messageRepository
   }
 
-  public async batchPickup(inboundConnection: InboundConnection) {
```

Comment:
Was the batchPickup message removed intentionally?

Reply:
It wasn't used because the `RecipientModule` directly creates a batch pickup message. I choose to remove this method, but it's probably better to use the service method instead.